src/Examples/Flowsheets/power_generation/rsofc/cpu.py

```diff
@@ -0,0 +1,720 @@
+##############################################################################
+# The development of this flowsheet/code is funded by the ARPA-E DIFFERENTIATE
+# project: “Machine Learning for Natural Gas to Electric Power System Design”
+# Project number: DE-FOA-0002107-1625.
+# This project is a collaborative effort between the Pacific Northwest National
+# Laboratory, the National Energy Technology Laboratory, and the University of
+# Washington to design NGFC systems with high efficiencies and low CO2
+# emissions.
+##############################################################################
+"""
+IDAES block for co2 purifucation process based on surrogate models
+
+This is a black box (surrogate based) model for CO2 purification process
+available in Aspen.
+
+The unit has 1 inlet stream for rich CO2 inlet: inlet
+the unti has 3 outlet streams: pureCO2, water, and vent
+
+The state variables are flow_mol, mol_fraction, temperature, and pressure.
+The state variables can be accessed via ports named: inlet, pure_CO2, water,
+and vent_gas
+
+The surrogate models were prepared varying rich inlet flowrate. Expected value
+11350 lbmol/hr, lower bound: 10850 lbmol/hr, and upper bound: 11850 lbmol/hr.
+300 samples have been developed using Latin Hypercube Sampling Method, the data
+set is trained by running the CPU rigorous model. ALAMO has been used to fit
+the surrogate models.
+Outputs are function of (inlet flowrate, inlet composition).
+This model 1 is simple and temperature and pressure are fixed
+
+pyomo block for co2 purifucation process based on surrogates
```

Comment:
This section seems disjointed, as if parts of it are missing. Notably this line starts without a capital which makes me thing something is missing.